Trying:
    import numpy as np
Expecting nothing
ok
Trying:
    from seawater import T68conv
Expecting nothing
**********************************************************************
File "seawater.py", line 37, in seawater.adtg
Failed example:
    from seawater import T68conv
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.adtg[1]>", line 1, in <module>
        from seawater import T68conv
    ImportError: cannot import name T68conv
Trying:
    T68conv  = 1.00024
Expecting nothing
ok
Trying:
    t = np.array([[ 0,  0,  0,  0,  0,  0], [10, 10, 10, 10, 10, 10], [20, 20, 20, 20, 20, 20], [30, 30, 30, 30, 30, 30], [40, 40, 40, 40, 40, 40]])
Expecting nothing
ok
Trying:
    t = t / T68conv
Expecting nothing
ok
Trying:
    s = np.array([[25, 25, 25, 35, 35, 35], [25, 25, 25, 35, 35, 35], [25, 25, 25, 35, 35, 35], [25, 25, 25, 35, 35, 35], [25, 25, 25, 35, 35, 35]])
Expecting nothing
ok
Trying:
    p = np.array([[0, 5000, 10000, 0, 5000, 10000], [0, 5000, 10000, 0, 5000, 10000], [0, 5000, 10000, 0, 5000, 10000], [0, 5000, 10000, 0, 5000, 10000], [0, 5000, 10000, 0, 5000, 10000]])
Expecting nothing
ok
Trying:
    adtg(s, t, p)
Expecting:
     array([[ 1.68710000e-05, 1.04700000e-04, 1.69426000e-04,
              3.58030000e-05, 1.17956500e-04, 1.77007000e-04],
            [ 1.00213255e-04, 1.60972016e-04, 2.06883149e-04,
              1.14904937e-04, 1.71376482e-04, 2.13000398e-04],
            [ 1.73853488e-04, 2.13558726e-04, 2.44501964e-04,
              1.84304853e-04, 2.21111157e-04, 2.49155462e-04],
            [ 2.41768241e-04, 2.64801063e-04, 2.82987774e-04,
              2.47979289e-04, 2.69501460e-04, 2.86177520e-04],
            [ 3.07934056e-04, 3.17039963e-04, 3.23045906e-04,
              3.09904786e-04, 3.18888326e-04, 3.24771901e-04]])
**********************************************************************
File "seawater.py", line 43, in seawater.adtg
Failed example:
    adtg(s, t, p)
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.adtg[7]>", line 1, in <module>
        adtg(s, t, p)
      File "seawater.py", line 78, in adtg
        T68 = T68conv * t
    NameError: global name 'T68conv' is not defined
Trying:
    s, pt, p = 40., 10., 4000.
Expecting nothing
ok
Trying:
    sw.alpha(s, pt, p)
Expecting:
    0.00025061316481624323
**********************************************************************
File "seawater.py", line 141, in seawater.alpha
Failed example:
    sw.alpha(s, pt, p)
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.alpha[1]>", line 1, in <module>
        sw.alpha(s, pt, p)
    NameError: name 'sw' is not defined
Trying:
    s, pt, p = 40.0, 10.0, 4000
Expecting nothing
ok
Trying:
    sw.aonb(s, pt, p)
Expecting:
    0.347650567047807
**********************************************************************
File "seawater.py", line 199, in seawater.aonb
Failed example:
    sw.aonb(s, pt, p)
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.aonb[1]>", line 1, in <module>
        sw.aonb(s, pt, p)
    NameError: name 'sw' is not defined
Trying:
    s, pt, p = 40.0, 10.0, 4000
Expecting nothing
ok
Trying:
    sw.beta(s,pt,p)
Expecting:
    0.00072087661741618932
**********************************************************************
File "seawater.py", line 269, in seawater.beta
Failed example:
    sw.beta(s,pt,p)
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.beta[1]>", line 1, in <module>
        sw.beta(s,pt,p)
    NameError: name 'sw' is not defined
Trying:
    n2, q, p_ave = sw.bfrq(s, t, p, lat)
Expecting nothing
**********************************************************************
File "seawater.py", line 363, in seawater.bfrq
Failed example:
    n2, q, p_ave = sw.bfrq(s, t, p, lat)
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.bfrq[0]>", line 1, in <module>
        n2, q, p_ave = sw.bfrq(s, t, p, lat)
    NameError: name 'sw' is not defined
Trying:
    t    = np.array([0, 10, 0, 10, 10, 30]) / T68conv
Expecting nothing
**********************************************************************
File "seawater.py", line 632, in seawater.cndr
Failed example:
    t    = np.array([0, 10, 0, 10, 10, 30]) / T68conv
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.cndr[0]>", line 1, in <module>
        t    = np.array([0, 10, 0, 10, 10, 30]) / T68conv
    NameError: name 'np' is not defined
Trying:
    p    = np.array([0, 0, 1000, 1000, 0, 0])
Expecting nothing
**********************************************************************
File "seawater.py", line 633, in seawater.cndr
Failed example:
    p    = np.array([0, 0, 1000, 1000, 0, 0])
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.cndr[1]>", line 1, in <module>
        p    = np.array([0, 0, 1000, 1000, 0, 0])
    NameError: name 'np' is not defined
Trying:
    s    = np.array([25, 25, 25, 25, 40, 40])
Expecting nothing
**********************************************************************
File "seawater.py", line 634, in seawater.cndr
Failed example:
    s    = np.array([25, 25, 25, 25, 40, 40])
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.cndr[2]>", line 1, in <module>
        s    = np.array([25, 25, 25, 25, 40, 40])
    NameError: name 'np' is not defined
Trying:
    sw.cndr(s, t, p)
Expecting:
    array([0.49800825, 0.65499015, 0.50624434, 0.66297496, 1.00007311, 1.52996697])
**********************************************************************
File "seawater.py", line 635, in seawater.cndr
Failed example:
    sw.cndr(s, t, p)
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.cndr[3]>", line 1, in <module>
        sw.cndr(s, t, p)
    NameError: name 'sw' is not defined
Trying:
    sw.cor(45)
Expecting:
    0.00010312445296824608
**********************************************************************
File "seawater.py", line 571, in seawater.cor
Failed example:
    sw.cor(45)
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.cor[0]>", line 1, in <module>
        sw.cor(45)
    NameError: name 'sw' is not defined
Trying:
    lat = np.array([0, 30, 45, 90])
Expecting nothing
**********************************************************************
File "seawater.py", line 433, in seawater.depth
Failed example:
    lat = np.array([0, 30, 45, 90])
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.depth[0]>", line 1, in <module>
        lat = np.array([0, 30, 45, 90])
    NameError: name 'np' is not defined
Trying:
    p   = np.array([[  500,   500,   500,  500], [ 5000,  5000,  5000, 5000], [10000, 10000, 10000, 10000]])
Expecting nothing
**********************************************************************
File "seawater.py", line 434, in seawater.depth
Failed example:
    p   = np.array([[  500,   500,   500,  500], [ 5000,  5000,  5000, 5000], [10000, 10000, 10000, 10000]])
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.depth[1]>", line 1, in <module>
        p   = np.array([[  500,   500,   500,  500], [ 5000,  5000,  5000, 5000], [10000, 10000, 10000, 10000]])
    NameError: name 'np' is not defined
Trying:
    sw.depth(p, lat)
Expecting:
    array([[  496.65299239,   495.99772917,   495.3427354 ,   494.03357499],
       [ 4915.04099112,  4908.55954332,  4902.08075214,  4889.13132561],
       [ 9725.47087508,  9712.6530721 ,  9699.84050403,  9674.23144056]])
**********************************************************************
File "seawater.py", line 435, in seawater.depth
Failed example:
    sw.depth(p, lat)
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.depth[2]>", line 1, in <module>
        sw.depth(p, lat)
    NameError: name 'sw' is not defined
Trying:
    s = np.array([[5, 10, 15, 20, 25, 30, 35, 40], [5, 10, 15, 20, 25, 30, 35, 40]])
Expecting nothing
**********************************************************************
File "seawater.py", line 1100, in seawater.fp
Failed example:
    s = np.array([[5, 10, 15, 20, 25, 30, 35, 40], [5, 10, 15, 20, 25, 30, 35, 40]])
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.fp[0]>", line 1, in <module>
        s = np.array([[5, 10, 15, 20, 25, 30, 35, 40], [5, 10, 15, 20, 25, 30, 35, 40]])
    NameError: name 'np' is not defined
Trying:
    p = np.array([[ 0, 0, 0, 0, 0, 0, 0, 0], [500, 500, 500, 500, 500, 500, 500, 500]])
Expecting nothing
**********************************************************************
File "seawater.py", line 1101, in seawater.fp
Failed example:
    p = np.array([[ 0, 0, 0, 0, 0, 0, 0, 0], [500, 500, 500, 500, 500, 500, 500, 500]])
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.fp[1]>", line 1, in <module>
        p = np.array([[ 0, 0, 0, 0, 0, 0, 0, 0], [500, 500, 500, 500, 500, 500, 500, 500]])
    NameError: name 'np' is not defined
Trying:
    sw.fp(s, p)
Expecting:
    array([[-0.27369757, -0.54232831, -0.81142026, -1.0829461, -1.35804594, -1.63748903, -1.9218401, -2.2115367], [-0.65010724, -0.91873798, -1.18782992, -1.45935577, -1.73445561, -2.01389869, -2.29824976, -2.58794636]])
**********************************************************************
File "seawater.py", line 1102, in seawater.fp
Failed example:
    sw.fp(s, p)
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.fp[2]>", line 1, in <module>
        sw.fp(s, p)
    NameError: name 'sw' is not defined
Trying:
    sw.grav(lat, z=0)
Expecting:
    9.8061898752053995
**********************************************************************
File "seawater.py", line 505, in seawater.grav
Failed example:
    sw.grav(lat, z=0)
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.grav[0]>", line 1, in <module>
        sw.grav(lat, z=0)
    NameError: name 'sw' is not defined
Trying:
    delt = np.array([15, 20, 5]) / T68conv  - 15
Expecting nothing
**********************************************************************
File "seawater.py", line 833, in seawater.salds
Failed example:
    delt = np.array([15, 20, 5]) / T68conv  - 15
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.salds[0]>", line 1, in <module>
        delt = np.array([15, 20, 5]) / T68conv  - 15
    NameError: name 'np' is not defined
Trying:
    rtx  = np.array([  1, 1.0568875, 0.81705885])**0.5
Expecting nothing
**********************************************************************
File "seawater.py", line 834, in seawater.salds
Failed example:
    rtx  = np.array([  1, 1.0568875, 0.81705885])**0.5
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.salds[1]>", line 1, in <module>
        rtx  = np.array([  1, 1.0568875, 0.81705885])**0.5
    NameError: name 'np' is not defined
Trying:
    sw.salds(rtx, delt)
Expecting:
    array([ 78.31921607,  81.5689307 ,  68.19023687])
**********************************************************************
File "seawater.py", line 835, in seawater.salds
Failed example:
    sw.salds(rtx, delt)
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.salds[2]>", line 1, in <module>
        sw.salds(rtx, delt)
    NameError: name 'sw' is not defined
Trying:
    r = np.array([1, 1.2, 0.65])
Expecting nothing
**********************************************************************
File "seawater.py", line 1032, in seawater.salrp
Failed example:
    r = np.array([1, 1.2, 0.65])
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.salrp[0]>", line 1, in <module>
        r = np.array([1, 1.2, 0.65])
    NameError: name 'np' is not defined
Trying:
    t = np.array([15, 20, 5]) / T68conv
Expecting nothing
**********************************************************************
File "seawater.py", line 1033, in seawater.salrp
Failed example:
    t = np.array([15, 20, 5]) / T68conv
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.salrp[1]>", line 1, in <module>
        t = np.array([15, 20, 5]) / T68conv
    NameError: name 'np' is not defined
Trying:
    p = np.array([0, 2000, 1500])
Expecting nothing
**********************************************************************
File "seawater.py", line 1034, in seawater.salrp
Failed example:
    p = np.array([0, 2000, 1500])
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.salrp[2]>", line 1, in <module>
        p = np.array([0, 2000, 1500])
    NameError: name 'np' is not defined
Trying:
    sw.salrp(r, t, p)
Expecting:
    array([1., 1.01694294, 1.02048638])
**********************************************************************
File "seawater.py", line 1035, in seawater.salrp
Failed example:
    sw.salrp(r, t, p)
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.salrp[3]>", line 1, in <module>
        sw.salrp(r, t, p)
    NameError: name 'sw' is not defined
Trying:
    t = np.array([15, 20, 5]) / T68conv
Expecting nothing
**********************************************************************
File "seawater.py", line 907, in seawater.salrt
Failed example:
    t = np.array([15, 20, 5]) / T68conv
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.salrt[0]>", line 1, in <module>
        t = np.array([15, 20, 5]) / T68conv
    NameError: name 'np' is not defined
Trying:
    sw.saltrt(t)
Expecting:
    array([ 1.,  1.11649272,  0.77956585])
**********************************************************************
File "seawater.py", line 908, in seawater.salrt
Failed example:
    sw.saltrt(t)
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.salrt[1]>", line 1, in <module>
        sw.saltrt(t)
    NameError: name 'sw' is not defined
Trying:
    t    = np.array([15, 20, 5]) / T68conv
Expecting nothing
**********************************************************************
File "seawater.py", line 752, in seawater.sals
Failed example:
    t    = np.array([15, 20, 5]) / T68conv
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.sals[0]>", line 1, in <module>
        t    = np.array([15, 20, 5]) / T68conv
    NameError: name 'np' is not defined
Trying:
    rt   = np.array([  1, 1.0568875, 0.81705885])
Expecting nothing
**********************************************************************
File "seawater.py", line 753, in seawater.sals
Failed example:
    rt   = np.array([  1, 1.0568875, 0.81705885])
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.sals[1]>", line 1, in <module>
        rt   = np.array([  1, 1.0568875, 0.81705885])
    NameError: name 'np' is not defined
Trying:
    sw.sals(rt,t)
Expecting:
    array([ 35.        ,  37.24562718,  27.99534701])
**********************************************************************
File "seawater.py", line 754, in seawater.sals
Failed example:
    sw.sals(rt,t)
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.sals[2]>", line 1, in <module>
        sw.sals(rt,t)
    NameError: name 'sw' is not defined
Trying:
    r = np.array([1, 1.2, 0.65])
Expecting nothing
**********************************************************************
File "seawater.py", line 970, in seawater.salt
Failed example:
    r = np.array([1, 1.2, 0.65])
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.salt[0]>", line 1, in <module>
        r = np.array([1, 1.2, 0.65])
    NameError: name 'np' is not defined
Trying:
    t = np.array([15, 20, 5]) / T68conv
Expecting nothing
**********************************************************************
File "seawater.py", line 971, in seawater.salt
Failed example:
    t = np.array([15, 20, 5]) / T68conv
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.salt[1]>", line 1, in <module>
        t = np.array([15, 20, 5]) / T68conv
    NameError: name 'np' is not defined
Trying:
    p = np.array([0, 2000, 1500])
Expecting nothing
**********************************************************************
File "seawater.py", line 972, in seawater.salt
Failed example:
    p = np.array([0, 2000, 1500])
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.salt[2]>", line 1, in <module>
        p = np.array([0, 2000, 1500])
    NameError: name 'np' is not defined
Trying:
    sw.salt(r, t, p)
Expecting:
    array([ 34.99999992,  37.24562765,  27.99534693])
**********************************************************************
File "seawater.py", line 973, in seawater.salt
Failed example:
    sw.salt(r, t, p)
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.6/doctest.py", line 1248, in __run
        compileflags, 1) in test.globs
      File "<doctest seawater.salt[3]>", line 1, in <module>
        sw.salt(r, t, p)
    NameError: name 'sw' is not defined
1 items had no tests:
    seawater
**********************************************************************
15 items had failures:
   2 of   8 in seawater.adtg
   1 of   2 in seawater.alpha
   1 of   2 in seawater.aonb
   1 of   2 in seawater.beta
   1 of   1 in seawater.bfrq
   4 of   4 in seawater.cndr
   1 of   1 in seawater.cor
   3 of   3 in seawater.depth
   3 of   3 in seawater.fp
   1 of   1 in seawater.grav
   3 of   3 in seawater.salds
   4 of   4 in seawater.salrp
   2 of   2 in seawater.salrt
   3 of   3 in seawater.sals
   4 of   4 in seawater.salt
43 tests in 16 items.
9 passed and 34 failed.
***Test Failed*** 34 failures.
